### PR TITLE
write tests for  reducers

### PR DESCRIPTION
**Pull request by theartisancodes**
#### What does this PR do?
Write tests for reducers
#### Description of Task to be completed?
N/A
#### Where should the reviewer start?
[Place link to file here]()
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[162438434](https://www.pivotaltracker.com/n/projects/2117172/stories/162438434)
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: src/reducers/loadingReducer.test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import * as ActionTypes from '../actions/actionTypes';
+import reducer from './loadingReducer';
+import initialState from './initialState';
+
+describe('Reducers :: Loading', () => {
+  const  getInitialState = initialState;
+  //let status = true;
```
Comment (andela-oakinseye) [inline src/reducers/loadingReducer.test.js]: We don't need this comment.

File: src/reducers/errorReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    const expected= getInitialState;
+    expect(reducer(undefined, action)).toEqual(expected);
+  });
+  it('should handle Error Actions', () => {
+    const action = { type: ActionTypes.ERROR_ACTION, error: {status: true}};
+    const expected = Object.assign({}, getInitialState.status, action.error.status);
+    expect(reducer(getInitialState, action)).toEqual(expected);
+  });
+  it('should handle FETCH_INCIDENT', () => {
+    const action = { type: ActionTypes.FETCH_INCIDENT, error: {status: true}};
```
Comment (andela-oakinseye) [inline src/reducers/errorReducer.test.js]: The style here should be consistent with line 24 and beyond.

File: src/reducers/incidentReducer.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import * as ActionTypes from '../actions/actionTypes';
+import reducer from './incidentReducer';
+import initialState from './initialState';
+
+describe('Reducers :: Incident', () => {
+const  getInitialState = initialState.incidents;
```
Comment (andela-oakinseye) [inline src/reducers/incidentReducer.test.js]: Too many white spaces after `const` keyword?

File: src/reducers/locationsReducer.test.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import reducer from './locationsReducer';
+import initialState from './initialState';
+
+describe('Reducers :: Locations', () => {
+  const  getInitialState = initialState.locations;
+
+  it('should get initial state by default', () => {
+    const action = { type: 'unknown' };
+    const expected= getInitialState;
+
```
Comment (andela-oakinseye) [inline src/reducers/locationsReducer.test.js]: Is there a reason for this line break?

File: src/reducers/rolesReducer.test.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import reducer from './rolesReducer';
+import initialState from './initialState';
+
+describe('Reducers :: Roles', () => {
+  const  getInitialState = initialState.roles;
+
+  it('should get initial state by default', () => {
+    const action = { type: 'unknown' };
+    const expected= getInitialState;
+
```
Comment (andela-oakinseye) [inline src/reducers/rolesReducer.test.js]: Line break?

File: src/reducers/staffReducer.test.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import reducer from './staffReducer';
+import initialState from './initialState';
+
+
+describe('Reducers :: Staff Reducer', () => {
+  const  getInitialState = initialState.staff;
+
+  it('should get initial state by default', () => {
+    const action = { type: 'unknown' };
+    const expected= getInitialState;
```
Comment (andela-oakinseye) [inline src/reducers/staffReducer.test.js]: Should add a space before the `=` sign

File: src/reducers/loadingReducer.test.js (hunk 1 above)
Comment (theartisancodes) [reply src/reducers/loadingReducer.test.js]: done

File: src/reducers/tokenReducer.test.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import * as ActionTypes from '../actions/actionTypes';
+import reducer from './tokenReducer';
+import initialState from './initialState';
+
+describe('Reducers :: Token Reducer', () => {
+  const  getInitialState = initialState;
+
+  it('should get initial state by default', () => {
+    const action = { type: 'unknown' };
+    const expected= getInitialState.hasToken;
```
Comment (andela-oakinseye) [inline src/reducers/tokenReducer.test.js]: Space before `=` sign?

File: src/reducers/incidentReducer.test.js (hunk 3 above)
Comment (theartisancodes) [reply src/reducers/incidentReducer.test.js]: done

File: src/reducers/locationsReducer.test.js (hunk 4 above)
Comment (theartisancodes) [reply src/reducers/locationsReducer.test.js]: its being accessed globally in the file just for clarity

**Review by andela-oakinseye — CHANGES_REQUESTED** [review]
Kindly fix the requested code changes.

**Review by theartisancodes — COMMENTED** [review]
done

File: src/reducers/rolesReducer.test.js (hunk 5 above)
Comment (theartisancodes) [reply src/reducers/rolesReducer.test.js]: done

File: src/reducers/locationsReducer.test.js (hunk 4 above)
Comment (theartisancodes) [reply src/reducers/locationsReducer.test.js]: done

File: src/reducers/errorReducer.test.js (hunk 2 above)
Comment (theartisancodes) [reply src/reducers/errorReducer.test.js]: done

**Review by andela-oakinseye — APPROVED** [review]

**Pull request merged**